examples/deployments/uc_functions/README.md

```diff
@@ -0,0 +1,27 @@
+# Unity Catalog Integration
+
+## Pre-requisites
+
+```bash
+pip install pandas openai databricks-sdk
+```
+
+## Running the example
+
+First, run the deployments server:
+
+```bash
+# Required to authenticate with Databricks
+export DATABRICKS_HOST="..."   # e.g. https://my.databricks.com
+export DATABRICKS_TOKEN="..."
+# Required to execute function in Databricks
+export DATABRICKS_WAREHOUSE_ID="..."
```

Comment:
Can we link to some docs showing users where to get the warehouse ID?

Reply:
added instructions on how to get it.